src/import/sst_service.rs

```diff
@@ -239,7 +243,11 @@ impl<Router: RaftStoreRouter> ImportSst for ImportSSTService<Router> {
 
         ctx.spawn(
             future
-                .map_err(Error::from)
+                .map_err(|e| {
+                    let err = Error::from(e);
+                    error_inc(&err);
```

Comment:
Could you move it `send_rpc_response!` ?

Reply:
[Done](https://github.com/tikv/tikv/pull/6395/files#diff-4da2eb18547c8073aba6b01f88a91120R33)